sycl/include/CL/sycl/backend/cuda.hpp

```diff
@@ -0,0 +1,32 @@
+//==---------------- cuda.hpp - SYCL CUDA backend --------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl/detail/defines.hpp>
+
+__SYCL_INLINE namespace cl {
```

Comment:
Following #1121, this needs to be changed to 
```suggestion
__SYCL_INLINE_NAMESPACE(cl) {
```

Reply:
see https://github.com/codeplaysoftware/sycl-for-cuda/pull/4